sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/EventHubsTimeoutException.cs

```diff
@@ -20,7 +20,7 @@ internal EventHubsTimeoutException(string message) : this(message, null)
         /// <param name="message"></param>
         /// <param name="innerException"></param>
         internal EventHubsTimeoutException(string message, Exception innerException)
-            : base(true, message, innerException)
+            : base(true, message, innerException, ErrorLevelType.UserError)
```

Comment:
I think this is misclassified.   According to the [docs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-messaging-exceptions):

`"The server did not respond to the requested operation within the specified time, which is controlled by OperationTimeout. The server may have completed the requested operation. This exception can happen due to network or other infrastructure delays."`

While the user may have set an unreasonable timeout, I'd expect the common case to be a transient issue with the network or service response time.